.github/workflows/build.yml

```diff
@@ -10,6 +10,9 @@ jobs:
   build:
     name: Build jupytergis
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
How do folks feel about testing only the oldest and newest versions we support? :thinking:

Reply:
Sounds good to me 👍🏽

Reply:
> How do folks feel about testing only the oldest and newest versions we support? 🤔

Sounds Good 👍🏻